R/utils.R

```diff
@@ -1,5 +1,21 @@
+#' HTTP Date String
+#'
+#' Given a POSIXct object, return a date string in the format required for a
+#' HTTP Date header. For example: "Wed, 21 Oct 2015 07:28:00 GMT"
+#'
+#' @noRd
+http_date_string <- function(time) {
+  lt <- as.POSIXlt(time, tz = "UTC")
+  weekdays <- c("Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun")
```

Comment:
Everything that I can find is using Sunday as the first day.

The current date of `"2022-06-02 11:15:24 EDT"` gives an answer of `"Fri, 02 Jun 2022 15:15:24 GMT"` which should be Thursday instead of Friday. 

[`base::weekdays()`](https://rdrr.io/r/base/weekday.POSIXt.html) gives Thursday. The [description of `%w` for weekday](https://rdrr.io/r/base/strptime.html) also has Sunday as the first day.

Using Sunday as the first day would also match: https://github.com/rstudio/httpuv/blob/220319d9ca2136cc5045148b1d7d57363e061be1/tests/testthat/helper-app.R#L118

```suggestion
  weekdays <- c("Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat")
```